test/integration/rpc.js

```diff
@@ -429,25 +429,10 @@ describe('Aepp<->Wallet', function () {
 
     it('Try to connect unsupported protocol', async () => {
       await expect(aepp.rpcClient.request(
-        METHODS.connect, {
-          name: 'test-aepp',
-          version: 2,
-          networkId: aepp.getNetworkId()
-        }
+        METHODS.connect, { name: 'test-aepp', version: 2 }
       )).to.be.eventually.rejectedWith('Unsupported Protocol Version').with.property('code', 5)
     })
 
-    it.skip('Try to connect unsupported network', async () => {
```

Comment:
Currently, wallet polls aepp with it's info. Aepp can connect to wallet after it gets wallet info. So, aepp can check that wallet is on the right network before connecting to it, otherwise message user about incomatibility or so. There are no ability for aepp to reject the connection (it just not establishing the connection), so nothing to test as I see.